modules/nf-core/chromap/chromap/tests/main.nf.test

```diff
@@ -123,8 +122,9 @@ nextflow_process {
         then {
             assertAll(
                 { assert process.success },
-                { assert snapshot(process.out.versions).match("versions") },
-                { assert snapshot(file(process.out.bam[0][1]).name).match() }
+                { assert snapshot(file(process.out.bam[0][1]).name,
+                                  process.out.versions).match()
+                }
```

Comment:
Can we use nft-bam?
```suggestion
                { assert snapshot(
                 bam(process.out.bam[0][1]).getHeaderMD5(),
                 bam(process.out.bam[0][1]).getReadsMD5()
                 process.out.versions).match()
                }
```

Reply:
Thanks @SPPearce !  I used now `nft-bam` as suggested 😄